velox/dwio/parquet/reader/RepeatedColumnReader.cpp

```diff
@@ -211,14 +211,6 @@ void MapColumnReader::read(
   elementReader_->seekTo(childTargetReadOffset_, false);
 }
 
-void MapColumnReader::filterRowGroups(
```

Comment:
I forgot the base class implementation is not suitable for these 2 classes.  Can we override the methods with empty implementations instead?

Reply:
Yes. Sorry not noticing that. Let me update. Thanks.

Reply:
Hi @yma11, just to confirm, do you see the failure when you run velox_parquet_e2e_filter_test? I wonder why the PR-time tests didn't catch this.

Reply:
Hi @kagamiori, I can't reproduce any failures in this suite. FYI.